packages/integration_test/.gitignore

```diff
@@ -0,0 +1,7 @@
+.DS_Store
```

Comment:
I think there is a global gitignore that will handle this

Reply:
baleeted